test/widgets/recent_dm_conversations_test.dart

```diff
@@ -172,16 +171,27 @@ void main() {
 
       void checkTitle(WidgetTester tester, String expectedText, [int? expectedLines]) {
         // TODO(#232): syntax like `check(find(…), findsOneWidget)`
-        final widget = tester.widget(find.descendant(
+        final matchedFinder = find.descendant(
           of: find.byType(RecentDmConversationsItem),
           // The title might contain a WidgetSpan (for status emoji); exclude
           // the resulting placeholder character from the text to be matched.
-          matching: findText(expectedText, includePlaceholders: false)));
+          matching: find.byWidgetPredicate(
+                (widget) {
+              if (widget is! Text) return false;
+              final textWidget = widget;
+              final fullText = textWidget.data ?? textWidget.textSpan?.toPlainText() ?? '';
+              return fullText.startsWith(expectedText);},
+            description: 'Text starting with "$expectedText"',),);
```

Comment:
Bump on https://github.com/zulip/zulip-flutter/pull/2019#discussion_r2607874950 : please do another round of self-review, on the whole PR, for indentation / code style